docs/docs/addons-shallow-renderer.md

```diff
@@ -0,0 +1,62 @@
+---
+id: shallow-renderer
+title: Shallow Renderer
+permalink: docs/shallow-renderer.html
+layout: docs
+category: Reference
+prev: test-utils.html
+next: animation.html
+---
+
+**Importing**
+
+```javascript
+import shallowRenderer from 'react-test-renderer/shallow'; // ES6
+var shallowRenderer = require('react-test-renderer/shallow'); // ES5 with npm
```

Comment:
nit: I think we might want to rename these to `ReactShallowRenderer` to reduce ambiguity between what's returned by the import (the class) and the instance you render with.

```js
import ReactShallowRenderer from 'react-test-renderer/shallow'; // ES6
var ReactShallowRenderer = require('react-test-renderer/shallow'); // ES5 with npm
```

I should have mentioned this before. Sorry.